test/commands/config_restore_e2e_test.go

```diff
@@ -109,7 +109,6 @@ func TestRestoreConfigs_FromDownloadWithPlatformManifestFile_withPlatformConfigs
 	subsetOfConfigsToDownload := "alerting-profile,management-zone"
 	suffixTest := "_download_automations"
```

Comment:
Not strictly related to this PR, why do we have the suffix `_download_automations`?

Reply:
I assume this was a copy&paste thing a while ago or the test got bigger over time (only-automation => everything) 🤔